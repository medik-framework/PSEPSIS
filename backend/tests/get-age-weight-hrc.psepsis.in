[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartScreening",
    "eventArgs": []
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [700],
    "waitFor":
     {
       "name": "Instruct",
       "args": ["get age"]
     }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "WeightEntered",
    "eventArgs": [15],
    "waitFor":
      {
        "name": "Instruct",
        "args": ["get weight"]
      }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "HighRiskConditionsEntered",
    "eventArgs": [0],
    "waitFor":
      {
        "name": "Instruct",
        "args": ["get high risk conditions"]
      }
  },
  {
    "action": "exit"
  }
]
