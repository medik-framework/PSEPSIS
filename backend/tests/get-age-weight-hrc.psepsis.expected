[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get age"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get weight"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmWeightEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get high risk conditions"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmHighRiskConditionsEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get 3 bucket measurements"
    ]
  }
]
